ingest/ingest-datasets.py

```diff
@@ -82,7 +82,7 @@ def create_document( self, entity ):
         if abstract:
             doc.update({"abstract": abstract})
 
-        publication_year = list(ds.objects(DCO.yearOfPublication))
+        publication_year = list(ds.objects(DCT.issued))
```

Comment:
That won't work for facets unless you want to modify the facet JS code for just this case.